src/orderbook/base.jl

```diff
@@ -0,0 +1,41 @@
+abstract type AbstractOrderBook{Tprice, Tvolume} <: AbstractPriceData{Tprice, Tvolume} end
+
+struct OrderBookException <: Exception
+    msg::String
+end
+
+function price(ob::AbstractOrderBook, volume::Volume; raise=false)
+    if volume == Volume(0)
```

Comment:
should probably be

    if volume == Tvolume(0)

Reply:
Building on my previous comment:

`if volume == zero(Real)` ?